telemetry/README.md

```diff
@@ -0,0 +1,23 @@
+# Glean Development Guide
+
+*This is a work in progress!*
+
+## Updating the `metric.yaml` file
+
+Currently, the source of truth for the shared `metrics.yaml` file lives [here](https://gist.github.com/kirill-demtchouk/41a38336b61b938833fe011bda4a2001) as a gist in Kirill's account. We need to figure out a better solution for this long-term but for the sake of getting everyone up to speed on where we are currently.
```

Comment:
could we elaborate who the parties are where this says "shared" file?

Reply:
good call! on it!